docs/source/models/gbd2017/causes/ischemic_stroke/ischemic_stroke.rst

```diff
@@ -0,0 +1,68 @@
+.. _2017_cause_ischemic_stroke:
+
+===================
+Ischemic Stroke
+===================
+
+Disease Description
+-------------------
+
+.. todo::
+
+   Add more information and references. In particular, find data about global prevalence and relation to disease fatal and non-fatal description.
+
+
+
+
+
+Modeling Ischemic Stroke in GBD 2017
+-------------------------------------------
+
+Strokes in GBD 2017
+++++++++++++++++++++++++++++++++++++++
+
+
+Modeling Strategy for Ischemic Stroke
+++++++++++++++++++++++++++++++++++++++++++++
+
+Cause Model Diagram
+--------------------------
+.. image:: ischemic_stroke_states.png
+  :width: 150
+
+Transitions Diagram
+--------------------------
+.. image:: ischemic_stroke_transitions.png
+  :width: 200
+
+Model Assumptions and Limitations
++++++++++++++++++
```

Comment:
I _think_ we're treating this one as it's own section (---- as opposed to +++), though I don't know if this has yet been discussed/standardized!

Reply:
That's where I've been putting it - as its own section, so it should be ------. I'm open to discussion if we want to change the organization, but that's what will match the current documents.

Reply:
I updated this to be its own section (-----). Thanks, y'all!